### ISSUE TITLE
":GoRun %" does not output anything if no newline at the end

### ISSUE THREAD

**Issue by tyru** [fatih/vim-go#1542]
### What did you do?

I did run `:GoRun %` while editing the following file:

```go
package main

import "os"

func main() {
  // ":GoRun %" of fatih/vim-go does not print any output
  bytes := []byte("Hello")
  // this does
  // bytes := []byte("Hello\n")
  os.Stdout.Write(bytes)
}
```

### What did you expect to happen?

Output `Hello`.
(I don't need to distinguish with/without newline. I just want to see what output by each line in command-line)

### What happened instead?

No output.

### Configuration

* vim version (major, minor, and included patches from `:version`): 8.0.1216
* vim-go version: v1.15 (96995056cbe744119ebd53e31068e713fa08db61)
* go version (`go version`): go version go1.9 linux/amd64
* operating system: Windows Subsystem Linux
* `vimrc` you used to reproduce: no configuration about vim-go

**Comment by Carpetsmoker** [comment]
I can't reproduce this on Linux.

Maybe try adding `os.Stdout.Sync()`?

**Comment by tyru** [comment]
@Carpetsmoker Hmm, same result here.

![default](https://user-images.githubusercontent.com/48169/32145901-2caaa61e-bd13-11e7-983d-63c660da0d7a.PNG)

**Comment by tyru** [comment]
Upgraded to latest version (5a3a2319a14f3e7dff7bca5af51d414dce92097d), but same result.

**Comment by bhcleek** [comment]
Is this still a problem for you? It's possible that it was resolved by #1900.

**Comment by bhcleek** [comment]
Closing due to lack of feedback.
